lms/static/sass/shared/_footer.scss

```diff
@@ -190,6 +190,11 @@
       }
     }
   }
+
+  @media print {
+    border: none;
+    padding: 10px 10px 0 10px;
```

Comment:
I just reviewed this locally, and it isn't clear to me what the border: 0 is resetting here. Additionally I would keep the default padding of 25x 10px 0 10px; eliminating the need for this media query in its entirety. 

If I'm wrong and the border is in fact resetting something, please move this up to line 12 so the media query is more clearly associated with its parent / container.